cli/running/running.go

```diff
@@ -126,6 +126,14 @@ type providerImpl struct {
 	instanceCtx *InstanceCtx
 }
 
+type SkipClusterConfig int
```

Comment:
This type name confuses me. It will affect not only cluster config but also instances scripts. I would use  for type name something more general, like `LoadMode` or  similar and specific name for certain enum values (see below).